configuration/appframework/scripts/getDateEnrolledByProgramUuid.js

```diff
@@ -0,0 +1,12 @@
+function getDateEnrolledByProgramUuid(patientPrograms, programUuid) {
+  if (!patientPrograms) {
+    return null;
+  }
+  var i, len = patientPrograms.length;
+  for (i = 0; i < len; ++i) {
+    if (patientPrograms[i]['programUuid'] === programUuid) {
+      return patientPrograms[i]['dateEnrolled'];
```

Comment:
What if the patient has multiple enrollments for the same program?  What happens then?  Could easily happen in the pregnancy program.  What is the use case for this function and can we make it more explicit?